pkg/release/release.go

```diff
@@ -0,0 +1,56 @@
+// Package release contains function to help finding things out about a given
+// embedded cluster release. It is being kept here so if we decide to manage
+// releases in a different way, we can easily change it.
+package release
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const (
+	baseURL = "https://github.com/replicatedhq/embedded-cluster"
```

Comment:
We should change this to a replicated.app endpoint, even if it just acts as a proxy for the moment